framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -119,6 +119,22 @@
     };
 
     /**
+     * Standard unchecked default locations that should be top
+     */
+    // Fields are default to top so that warnings are issued at field reads, which we believe are more common
+    // than field writes. Future work is to specify different defaults for field reads and field writes.
```

Comment:
Add: "To also catch all errors on field assignments, the bottom type should be used for assignments." or something like that.